src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/GetTimelineTest.kt

```diff
@@ -278,7 +278,6 @@ class GetTimelineTest : IntegrationTestBase() {
           it.hasEventType(TimelineEventType.INDUCTION_CREATED)
             .hasPrisonId("BXI")
             .hasSourceReference(induction.reference.toString())
-            .hasNoContextualInfo() // creating an induction has no contextual info
```

Comment:
Is it worth replacing this with a call to `.hasContextualInfo` to assert it has the correct/expected contextual info map?

Reply:
can do/ was being lazy 😂